tex/measure/martingale.tex

```diff
@@ -259,7 +259,7 @@ \section{Supermartingales}
 \prototype{Visiting a casino is a supermartingale, assuming house odds.}
 
 \begin{definition}
-	Let $X_0$, $X_1$, \dots be a sequence of random variables
+	Let $X_0$, $X_1$, \dots\ be a sequence of random variables
```

Comment:
Not sure if this is desirable, looking at it more there are quite a few places that use the other notation (without `\` after `\dots`).

Reply:
If `\` is missing after `\dots` that's actually just a mistake, at some point I might do a regex search to patch them all.